src/main/java/com/google/jenkins/plugins/credentials/oauth/RemotableGoogleCredentials.java

```diff
@@ -136,7 +137,7 @@ public Credential getGoogleCredential(
   /**
    * The time at which the accessToken will expire.
    */
-  private final DateTime expiration;
+  private final long expiration;
```

Comment:
Maybe you also want to rename the field just to cause there are usages outside the case we know about